packages/default-join-server/.github/workflows/ci.yaml

```diff
@@ -1,3 +1,4 @@
+# NOTE: this file is from the separate repo era and is NOT used (so probably outdated). TODO: delete?
```

Comment:
This file should be deleted once the functionality is integrated to main CI build.

Reply:
I wasn't sure to what extent it was... but sure, I'll delete it.